pkg/neorpc/result/peer_heights.go

```diff
@@ -0,0 +1,64 @@
+package result
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+type (
+	GetPeerHeights struct {
+		PeerHeights PeerHeights `json:"peerHeights"`
+	}
+
+	PeerHeights []PeerHeight
+
+	PeerHeight struct {
+		Address   string `json:"address"`
+		UserAgent string `json:"userAgent"`
+		Height    uint32 `json:"height"`
+	}
+)
+
+// NewGetPeerHeights creates a new GetPeerHeights structure.
+func NewGetPeerHeights() GetPeerHeights {
+	return GetPeerHeights{
+		PeerHeights: []PeerHeight{},
+	}
+}
+
+func (g *GetPeerHeights) AddPeers(ps []PeerHeight) {
+	g.PeerHeights = append(g.PeerHeights, ps...)
+}
+
+func (p *PeerHeight) UnmarshalJSON(data []byte) error {
```

Comment:
Not sure why you need it.

Reply:
`UnmarshalJSON` you mean? I followed `getpeers` as an example and just assumed I had to implement it. Is it only necessary to have a custom unmarshaller in specific situations? I guess when C# has the method and the default Go JSON unmarshalling would be incorrect?

Reply:
Embedded JSON comments should deal with marshalling, so likely this code can be removed.

Reply:
> Is it only necessary to have a custom unmarshaller in specific situations?

Only when JSON tags can't do the thing. And this case is rather simple, so you don't need a custom unmarshaller.